content/en/apps/guides/security/securely-onboarding-users-at-scale.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Securely onboarding users at scale"
+linkTitle: "Securely Onboarding Users"
+weight:
+description: >
+    How to securely create users, handle password changes and password breaches
+relatedContent: >
+    apps/guides/training/onboarding
+    apps/guides/database/couchdb-authentication
+    apps/tutorials/contact-and-users-1
+    apps/guides/data/users-bulk-load
+    apps/concepts/users
+    apps/concepts/access/#magic-links-for-logging-in-token-login
+
+---
+
+{{% pageinfo %}}
+The more secure a system is, the harder it is to use. Following this document will achieve the highest level of security. That said, implementers need to know when ease of use is more important than a more secure system. By reading this document you should be able to know when to make the "more secure" vs "easier to use" trade off. 
+
+No system is perfectly secure - be prepared to remediate a security breach!
+{{% /pageinfo %}}
+
+When a CHT deployment will support 100s or even 1000s of users, secure credential management becomes critical. Assuming there's a 10:1 ratio of 4-person families per CHW, this means there's potentially over 40,000 patients in a large CHT instance. Their data is sensitive and should never be viewed, or worse edited, by any unauthorized parties. By following best practices, being aware of and prepared for a worst case scenario of a password breach, patient data can be kept safe and CHWs can be kept online and able to deliver care.
```

Comment:
The patient calculation was based on a few assumptions and I don't think it's necessary.

```suggestion
When a CHT deployment will support hundreds of users or more, secure credential management becomes critical. Their data is sensitive and should never be viewed, or worse edited, by any unauthorized parties. By following best practices, being aware of and prepared for a worst case scenario of a password breach, patient data can be kept safe and CHWs can be kept online and able to deliver care.
```